server_installation/topics/operator/keycloak-user-cr.adoc

```diff
@@ -0,0 +1,76 @@
+
+=== KeycloakUser Custom Resource
+
+{project_operator} allows application developers to represent {project_name} Users as Custom Resources:
+
+.`KeycloakUser` Custom Resource
+```yaml
+apiVersion: keycloak.org/v1alpha1
+kind: KeycloakUser
+metadata:
+  name: example-realm-user
+spec:
+  user:
+    username: "realm_user"
+    firstName: "John"
+    lastName: "Doe"
+    email: "user@example.com"
+    enabled: True
+    emailVerified: False
+    realmRoles:
+      - "offline_access"
+    clientRoles:
+      account:
+        - "manage-account"
+      realm-management:
+        - "manage-users"
+  realmSelector:
+    matchLabels:
+      app: sso
+```
+
+TIP: Note, that `realmSelector` needs to match labels of an existing `KeycloakRealm` Custom Resource.
```

Comment:
Change this three lines into `IMPORTANT: ` annotation, so they aren't overlooked.

Reply:
Fixed.